### DISCUSSION TITLE
Welcome to Quantum-Codes Discussions!

### DISCUSSION THREAD

**Discussion by Quantum-Codes** [Quantum-Codes/Ankit_Anmol#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you.
  *As profile comments. (The only point I added)
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below your scratch profile(if any) and tell and which planet u live in

**Comment by Quantum-Codes** [comment]
I'm on mars

**Reply by Sid72020123** [reply]
Lol

**Reply by Quantum-Codes** [reply]
you?

**Reply by TheBugCoder** [reply]
Why did you leave Scratch?

**Reply by Sid72020123** [reply]
No he didn't leaved Scratch. He is banned :(

**Reply by Quantum-Codes** [reply]
> Why did you leave Scratch?

Im banned for 3 months :/ so sad...

**Reply by Sid72020123** [reply]
Hi Ankit_Anmol. You need to update the studio code of your library because the Studio API is changed :(
I already did that in my library

**Reply by Quantum-Codes** [reply]
Oh ok

**Reply by Quantum-Codes** [reply]
1 change only? Host id

**Reply by Sid72020123** [reply]
Yes lol

**Reply by Quantum-Codes** [reply]
> Yes lol

Online? How to search username by ID?

**Reply by Sid72020123** [reply]
I haven't found a way yet :(

**Reply by Sid72020123** [reply]
Bye I am busy

**Reply by Quantum-Codes** [reply]
Bye

**Reply by Sid72020123** [reply]
But you can use an api to get the managers of a studio! Its in ScratchConnect! Bye! Good Night!

**Reply by Quantum-Codes** [reply]
> But you can use an api to get the managers of a studio! Its in ScratchConnect! Bye! Good Night!

I'll see later! If u wanna talk next time here, pls create a new topic lol

**Reply by Sid72020123** [reply]
Yes LOL It's a little confusing here. Also ScraGet should get some features that are without LOGIN such as get user's projects as in ScratchConnect.

**Reply by Quantum-Codes** [reply]
I don't have time :(

**Reply by Quantum-Codes** [reply]
Male an issue there, I will then try lol

**Comment by Sid72020123** [comment]
What?

**Reply by Quantum-Codes** [reply]
Planet xD

**Reply by Quantum-Codes** [reply]
> What?
> […](#)
> On Wed, 7 Jul 2021, 6:59 pm Ankit_Anmol, ***@***.***> wrote: you? — You are receiving this because you commented. Reply to this email directly, view it on GitHub <[#1 (reply in thread)](https://github.com/Quantum-Codes/Ankit_Anmol/discussions/1#discussioncomment-974894)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AQX7QPRTHAGRS2QZ2FE7YELTWRJEXANCNFSM4756TWCQ> .

@Sid72020123  how do I contribute to wiki of scratchclient?

**Reply by Sid72020123** [reply]
Fork it and create a pull request

**Reply by Quantum-Codes** [reply]
> Fork it and create a pull request

I want to contribute to wiki. If i fork, it doesn't fork the wiki

**Comment by Sid72020123** [comment]
Oh! I don't know how to do that!

**Reply by Quantum-Codes** [reply]
I will ask there itself. I said CubeyTheCube to make a discussion

**Comment by Sid72020123** [comment]
Yes! I saw that!

**Reply by Quantum-Codes** [reply]
:)

**Comment by Sid72020123** [comment]
The Scratch API library is finally released! Go to
https://github.com/Sid72020123/scratchconnect#scratchconnect-v10 to check
it out!

**Reply by Quantum-Codes** [reply]
Great!

**Comment by Sid72020123** [comment]
Can you invite me to the scratch profile comments repo

**Comment by Sid72020123** [comment]
I also made two bots using scratchconnect: https://scratch.mit.edu/users/JokeBot/ and https://scratch.mit.edu/users/FactBot/

**Reply by Quantum-Codes** [reply]
I saw. What can collaboratorsdo

**Reply by Sid72020123** [reply]
I dont know

**Reply by Quantum-Codes** [reply]
Umm

**Reply by Quantum-Codes** [reply]
I will invite I 1 min

**Reply by Sid72020123** [reply]
Ok

**Comment by ninjamar** [comment]
hello

**Reply by Quantum-Codes** [reply]
Hi!

**Reply by ninjamar** [reply]
what are u making?

**Reply by Quantum-Codes** [reply]
Nothing much because of exams. Doing now:

1. ScraGet package updates
2. Some API tracker for a game
3. Testing wasteof.money API

**Reply by ninjamar** [reply]
what is the wasteof.money api?

**Reply by Quantum-Codes** [reply]
https://github.com/wulliy/wasteof2-docs :)

**Reply by ninjamar** [reply]
thanks

**Reply by Quantum-Codes** [reply]
Welcome. What are you making btw?

**Reply by ninjamar** [reply]
i'm making my website. i almost have the blog done

**Reply by Quantum-Codes** [reply]
Ok

**Reply by Quantum-Codes** [reply]
Your rickroll doesn't load

**Reply by ninjamar** [reply]
bye

**Reply by ninjamar** [reply]
i'm not sure why. the url of the video is fine

**Reply by Quantum-Codes** [reply]
I'll try checking later

**Reply by ninjamar** [reply]
thanks

**Reply by ninjamar** [reply]
i fixed it by using an iframe. chrome doesn't allow autoplay

**Reply by Quantum-Codes** [reply]
Oh ok

**Reply by Quantum-Codes** [reply]
You can embed a turbowarp embed. It does autoplay but without sound.
Chiroyce has a rickroll project

**Reply by ninjamar** [reply]
i think iframe works for now

**Reply by Quantum-Codes** [reply]
Ok

**Comment by SparkScratch-P** [comment]
## Hi! Ankit.
## SOrry for the late response. I saw your message today. I will surely try to help u out!

**Reply by Quantum-Codes** [reply]
Thank you :)

**Reply by SparkScratch-P** [reply]
![image](https://user-images.githubusercontent.com/73777108/151338802-4e8532c7-ee04-4287-98a5-dab14c6f11df.png)
 ## Did you mean something like this?

**Reply by Quantum-Codes** [reply]
Not exactly 50%. I meant the vertical divider should be towards the left just as the main table

**Reply by SparkScratch-P** [reply]
OK, I understand. Can u please give me the link to the Repo? I would work on it thn!

**Reply by Quantum-Codes** [reply]
I have it on replit https://replit.com/@TwilightClub/TwilightClub 
The HTML file is in templates folder and CSS file in static folder. (All other files are used for getting and maintaining data stuff)

**Reply by SparkScratch-P** [reply]
I am not used to with replit, can u pls make  a git repo here? I will surely like to work!

**Reply by Quantum-Codes** [reply]
https://github.com/Quantum-Codes/help I am using python to fill up the data fields stuff. So, I have put some placeholder values. https://quantum-codes.github.io/help/
(Sorry for the weird indentation)

**Reply by SparkScratch-P** [reply]
Done Bro! Check out and merge! And, No 'sorry'... more weird= more challenges, and I love challenges!

**Reply by Quantum-Codes** [reply]
Lol ok

**Comment by ghost** [comment]
Did you actually get banned on Scratch?

**Reply by Quantum-Codes** [reply]
Yes :(

**Reply by ghost** [reply]
What happened, why'd you get banned?

**Reply by SparkScratch-P** [reply]
The administration is so bad at Scratch that it merely needs a reason to ban you.... they will just ban you, and never look back at your mails!

**Reply by Quantum-Codes** [reply]
> What happened, why'd you get banned?

@SparkScratch-P is correct, they claim to send me emails and read it but I didn't get anything.

I was banned for sharing an already public email of another Scratcher. I get it. It's my fault. But now it's been like 1 year

**Reply by noverd** [reply]
Ok

**Reply by ghost** [reply]
@Quantum-Codes Oh. 😞 Did you try to send an appeal and explain that it was already public?

**Comment by noverd** [comment]
Hi, Quantum-Codes!
Thanks for your project - https://github.com/Quantum-Codes/Scratch-Profile-Comments
I'm developing my wrapper API for scratch and your project has been very helpful! I will leave you thanks in ReadMe.md within 2 days

https://github.com/noverd/ScratchPlus

**Reply by Quantum-Codes** [reply]
Glad it helped!
